include/cantera/kinetics/Arrhenius.h

```diff
@@ -195,6 +198,97 @@ class ArrheniusRate final : public ArrheniusBase
 };
 
 
+//! Electron temperature reaction rate type depends only on both
+//! temperature and electron temperature.
+/*!
+ * The form of the electron temperature reaction rate coefficient is similar to an
+ * Arrhenius reaction rate coefficient. The temperature exponent (b) is applied to
+ * the electron temperature instead. In addition, the exponential term with
+ * activation energy for electron is included.
+ *
+ *   \f[
+ *        k_f =  A T_e^b \exp (-Ea_T/RT) \exp (-Ea_{T_e}/RT_e)
+ *   \f]
+ *
+ * @ingroup arrheniusGroup
+ */
+class ETempRate final : public ArrheniusBase
```

Comment:
Declaring this class as `final` here is the correct approach (`BlowersMaselRate` doesn't do this, but should).